src/wallet/spend.h

```diff
@@ -180,15 +180,22 @@ std::optional<SelectionResult> SelectCoins(const CWallet& wallet, CoinsResult& a
                                            const CAmount& nTargetValue, const CCoinControl& coin_control,
                                            const CoinSelectionParams& coin_selection_params) EXCLUSIVE_LOCKS_REQUIRED(wallet.cs_wallet);
 
-struct CreatedTransactionResult
+struct FundedTransactionResult
```

Comment:
What about naming it `TransactionDetails` or something similar.

The double "result" wording in stuff like `Util::Result<FundedTransactionResult>` sounds redundant.

(for the other one, `CreatedTransactionResult` could do the same and remove the "result" wording too)

Reply:
Removing the redundancy seems to make sense. Not sure about `TransactionDetails` -- if one of the two structs is named like this, what would be the name of the other? (`{More,Less}TransactionDetails`? :stuck_out_tongue: )